Asgardia.Constitution.Proposal.txt

```diff
@@ -75,7 +75,7 @@ Article 4. Supreme Values of Asgardia
  l. fairness, and freedom;
  m. harmonious existence of the individual, society, and the Space Nation.
 § 5. Asgardia’s Supreme Values are equally important and equally implemented in state administration. In the event of a conflict between the Supreme Values, the absolute Supreme Value of Asgardia prevails.
-§ 6. Asgardia’s Supreme Values apply to relations with Earth nation states and citizens, and any other non-Asgardian territories in space, as well as any discovered alien intelligence.
+§ 6. Asgardia’s Supreme Values apply to relations with Earth nation states and citizens, and any other non-Asgardian territories and peoples in space.

```

Comment:
Amusingly, this is the one I was struggling with trying to figure out how to phrase. I tried to capture this with "peoples", but I know that might not get the spirit of the message.